### PR TITLE
added ADB-PLUGIN

### PR DESCRIPTION
**Pull request by vinit-ramon**

### CODE REVIEW

**Review by spanchum — APPROVED** [review]
looks good

**Review by spanchum — APPROVED** [review]

File: adb-grafana-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# About Autonomous Database plugin for Grafana
+
+## Introduction
+The plugin connects to Oracle Autonomous Database and make queries to it and display them on Grafana.
+
+It is a datasouce backend plugin which requires username, password and basic connection string to connect to Oracle ADB.
```
Comment (SantoshKChaudhari) [inline adb-grafana-plugin/README.md:6]: typo - datasource

File: adb-grafana-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+It can query for timeseries and Non-timeseries data. 
+
+The Plugin is compatible with grafana version >=9.1.2.
+
+
+## Installation
+First Download the Plugin Zip file from https://bitbucket.oci.oraclecorp.com/users/vivinitk/repos/oracle-adb-grafana-plugin/browse and Extract it. Let's say the extracted path is $plug_path and it contains oracle-adb-datasource folder along with this README and version file.
```
Comment (SantoshKChaudhari) [inline adb-grafana-plugin/README.md:14]: remove internal bit bucket link reference

File: adb-grafana-plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* You can query for `non-timeseries` data too to show it on table.
+
+*   ![Config page](src/img/query2.png)
+
+* Save the Panel to your Dashboard. 
+
+
+
+## Code link
+https://bitbucket.oci.oraclecorp.com/users/vivinitk/repos/oracle-oracleadb-datasource/browse/README.md?at=refs%2Fheads%2Fnewplug
```
Comment (SantoshKChaudhari) [inline adb-grafana-plugin/README.md:70]: remove the internal link reference and use the public link

File: adb-grafana-plugin/dist/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+It is a datasouce backend plugin which requires username, password and basic connection string to connect to Oracle ADB.
+
+It can query for timeseries and Non-timeseries data. 
+
+The Plugin is compatible with grafana version >=9.1.2.
+
+
+## Installation
+First Download the Plugin Zip file from https://bitbucket.oci.oraclecorp.com/users/vivinitk/repos/oracle-adb-grafana-plugin/browse and Extract it. Let's say the extracted path is $plug_path and it contains oracle-adb-datasource folder along with this README and version file.
```
Comment (SantoshKChaudhari) [inline adb-grafana-plugin/dist/README.md:14]: remove internal reference

File: adb-grafana-plugin/src/README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+It is a datasouce backend plugin which requires username, password and basic connection string to connect to Oracle ADB.
+
+It can query for timeseries and Non-timeseries data. 
+
+The Plugin is compatible with grafana version >=9.1.2.
+
+
+## Installation
+First Download the Plugin Zip file from https://bitbucket.oci.oraclecorp.com/users/vivinitk/repos/oracle-adb-grafana-plugin/browse and Extract it. Let's say the extracted path is $plug_path and it contains oracle-adb-datasource folder along with this README and version file.
```
Comment (SantoshKChaudhari) [inline adb-grafana-plugin/src/README.md:14]: remove internal link reference

**Review by SantoshKChaudhari — CHANGES_REQUESTED** [review]

**Review by SantoshKChaudhari — APPROVED** [review]

**Pull request merged**